docs/src/routes/docs/components/typography.svx

```diff
@@ -0,0 +1,89 @@
+---
+name: Typography
+---
+
+<script>
+  import { H1, H2, H3, Headline, Subhead, Label } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Typography
+
+Typography elements with embedded responsive styles.
+
+Also comes with a `<TypographyStyles>` component, which only adds global styles, setting the `$font` for the whole page, coloring links, etc. A good place to put it for Sapper, for example, would be anywhere in the main [layout file](https://sapper.svelte.dev/docs#Layouts).
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Headline>Main stuff!</Headline>
+    <Subhead>Make sure you don't miss out!</Subhead>
+
+    <H1>Heading 1</H1>
+    <H2>Heading 2</H2>
+    <H3>Heading 3</H3>
+
+    <Label>label</Label>
+    <Label small>smaller label</Label>
+  </div>
+  <div slot="source">
+
+```svelte
+<Headline>Main stuff!</Headline>
+<Subhead>Make sure you don't miss out!</Subhead>
+
+<H1>Heading 1</H1>
+<H2>Heading 2</H2>
+<H3>Heading 3</H3>
+
+<Label>label</Label>
+<Label small>smaller label</Label>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+_This property only applies to `<Label>`_.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`small`** | `false` | `boolean` | Makes the `<Label>` smaller. |
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`...`** |  | `any` | Everything will be passed to the underlying element. |
+
+### Class Properties
+
+_The following only applies to `<Headline>`, `<Subhead>` and `<Label>` components._
+
+All `<Headline>` components use the `.headline` class, all `<Subhead>` components use the `.subhead` class, all `<Label>` components use the `.label` class.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`class`** | `null` | `string` | A class string to add to the component. |
+
+
+## Slots {#slots}
+
+### Default slot
+
+The content of the typographic element.
+
+
+## SCSS Variables {#scss-variables}
+
+| Name | Description | Default |
+| ---- | ----------- | ------- |
+| **`$main`** | The color of some headings, the headline and the label. <br /> When `<TypographyStyles>` are applied, also colors the links. | <ColorPreview value="#4300B0" /> |
+| **`$link-visited`** | When `<TypographyStyles>` are applied, colors the visited links. | <ColorPreview value="#50005F" /> |
+| **`$bold-font-weight`** | The bold font weight of headings. <br /> When `<TypographyStyles>` are applied, also adjusts the boldness of `<strong>`. | `500` |
+| **`$thin-font-weight`** | The thin font weight of the subhead. | `300` |
+| **`$font`** | When `<TypographyStyles>` are applied, becomes the font for the entire page. | `'Ubuntu'` |
```

Comment:
As mentioned in a previous comment, it is not clear _"When `<TypographyStyles>` are applied"_ really means

Reply:
Idk how we can improve this